### PR TITLE
feat (api, ui, cli): refactor status & Admin UI

### PR DESCRIPTION
**Pull request by yesnault**
Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

**Comment by yesnault** [comment]
CLI:
capture d ecran 2017-12-17 a 22 48 59

Web UI
capture d ecran 2017-12-17 a 23 18 47

File: ui/src/app/views/admin/status/status.component.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    selector: 'app-status',
+    templateUrl: './status.html',
+    styleUrls: ['./status.scss']
+})
+export class StatusComponent implements OnInit {
+
+    status: MonitoringStatus;
+
+    constructor(private _monitoringService: MonitoringService) {
+        this._monitoringService.getStatus()
```
Comment (bnjjj) [inline ui/src/app/views/admin/status/status.component.ts]: put a loading

File: ui/src/app/views/admin/status/status.component.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            .subscribe(r => {
+                this.status = r;
+            });
+    }
+
+    ngOnInit() {
```
Comment (bnjjj) [inline ui/src/app/views/admin/status/status.component.ts]: to delete

File: ui/src/app/views/admin/status/status.component.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import {MonitoringService} from '../../../service/monitoring/monitoring.service';
+import { forEach } from '@angular/router/src/utils/collection';
+import { MonitoringStatus } from 'app/model/monitoring.model';
+
+@Component({
+    selector: 'app-status',
+    templateUrl: './status.html',
+    styleUrls: ['./status.scss']
+})
+export class StatusComponent implements OnInit {
```
Comment (bnjjj) [inline ui/src/app/views/admin/status/status.component.ts]: delete useless OnInit

**Pull request merged**